python/ray/data/_internal/logical/operators/input_data_operator.py

```diff
@@ -10,37 +10,25 @@
 class InputData(LogicalOperator):
     """Logical operator for input data.
 
-    This may hold cached blocks from a previous Dataset execution, or
-    the arguments for read tasks.
+    This may hold cached blocks from a previous Dataset execution.
```

Comment:
Drive-by change to correct this docstring. `read_xxx` APIs use the `Read` logical operator, not `InputData`.